open_lm/train.py

```diff
@@ -228,11 +236,37 @@ def train_one_epoch(model, data, loss, epoch, step, optimizer, scaler, scheduler
                             * inputs_ii.shape[0]
                             / inputs.shape[0]
                         )
+                    if args.moe_freq > 0:
+                        moe_args = MoEArgs(
```

Comment:
Can we construct moeargs outside of the train loop to make the training loop smaller and avoid constructing this every batch?

Reply:
done